docs/meetings/team/2022_11_29.md

```diff
@@ -0,0 +1,58 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:30am
+
+**End Time** 10:40am
+
+**Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Matt
+2. Veronica
+3. Logan
+4. Edouard
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad joke of the day
+
+## Announcements
+
+- GitStats are due Friday
+- Peer testing report is due (next) Friday
+  - Prepare for the report this week for peer testing
+- Peer Testing on Wed and Friday
+
+## Last Week (Restrospective)
+
+- Feedback from usability tasks
+  - TA really liked our tasks
+  - We should have more working cases
+    - Should make questions for user testing
+- Unit tests
+  - 3 pass, expect corrupt wav file case fails
+
+## This Week  
+
+### Peer Testing
+
+- Peer testing is complete, made a release for peer testing
+- Have the peers use the Kukerella's front-end:
+  - If there is more time then the user should create audio files from the site, they would re-name the file
+- Including questions about the video and voice-synch
+- All cases have been tested!
+
+## Client Meeting
+
+- Not sure...
```

Comment:
It'll either be this Friday or next Friday - but December 8 is the last day of classes and I don't want anyone stressed out in case they have an exam on the weekend, so I think this Friday would be best.

As for what we'll go over, we can discuss points below (see other comments)

Reply:
If we're sticking to bi-weekly, this would be the biweekly point so maybe this week would be good